lib/layouts/Sidebar.js

```diff
@@ -98,17 +121,30 @@ const Sidebar = ({
 
       <div className="nui-sidebar__footer">
         {profileInfo && (
-          <div className="nui-sidebar__profile-wrapper">
-            <div className="nui-sidebar__profile">
-              <div className="nui-sidebar__avatar">JM</div>
-              {!collapsed && (
-                <div className="nui-sidebar__profile-info">
-                  <h2>{profileInfo.name}</h2>
-                  <p>{profileInfo.email}</p>
-                </div>
-              )}
-            </div>
-          </div>
+          <Dropdown
+            customTarget={() => (
+              <ProfileSection
+                collapsed={collapsed}
+                profileInfo={profileInfo}
+                onClick={() => {
+                  setIsProfileDropdownOpen(!isProfileDropdownOpen);
+                }}
+              />
+            )}
+            isOpen={isProfileDropdownOpen}
+            position={collapsed ? "bottom-end" : "bottom"}
+            buttonProps={{
+              icon: Down,
+            }}
+            closeOnSelect={false}
+            closeOnOutsideClick={true}
+          >
+            {profileInfo?.dropdownProps?.map((item, idx) => (
```

Comment:
profileInfo condition is already checked.
```suggestion
            {profileInfo.dropdownProps?.map((item, idx) => (
```

Reply:
Fixed